searx/engines/npm.py

```diff
@@ -46,28 +44,23 @@ def response(resp):
     content = resp.json()
     for entry in content["results"]:
         package = entry["package"]
-        url = package["links"]["npm"]
-        title = package["name"]
-        content = package.get("description")
-        version = package.get("version")
-        maintainer = package.get("author", {}).get("name")
         publishedDate = package.get("date")
+        if publishedDate:
+            publishedDate = parser.parse(publishedDate)
         tags = list(entry.get("flags", {}).keys()) + package.get("keywords", [])
-        homepage = package["links"].get("homepage")
-        source_code_url = package["links"].get("repository")
         results.append(
             {
                 "template": "packages.html",
-                "url": url,
-                "title": title,
-                'package_name': title,
-                "content": content,
-                "version": version,
-                "maintainer": maintainer,
-                'publishedDate': parser.parse(publishedDate),  # type: ignore
+                "url": package["links"]["npm"],
+                "title": package["name"],
+                'package_name': package["name"],
+                "content": package.get("description", ""),
```

Comment:
Field `content` must not be `None` / at least an empty string is needed to show the result in the result list.